examples/k8s-patch-basic.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: k8s-patch-basic-
+  labels:
+    workflows.argoproj.io/test: "true"
+  annotations:
+    workflows.argoproj.io/description: |
+      This example shows a standard patch with the default mergeStrategy
```

Comment:
Add what the default mergeStrategy is here

Reply:
Added in #13317